packages/react-magma-dom/src/components/Pagination/SimplePagination.tsx

```diff
@@ -0,0 +1,190 @@
+    font-weight: 500;
+  }
+`;
+
+export const SimplePagination = React.forwardRef<
+  HTMLDivElement,
+  PaginationProps
+>((props, ref) => {
+  const {
+    count,
+    defaultPage,
+    disabled,
+    hideNextButton,
+    hidePreviousButton,
+    id: defaultId,
+    isInverse,
+    testId,
+    onPageChange,
+    ...other
+  } = props;
+  const theme = React.useContext(ThemeContext);
+
+  const i18n = React.useContext(I18nContext);
+
+  const id = useGenerateId(defaultId);
+
+  let [selectedPage, setSelectedPage] = React.useState(
+    defaultPage ? defaultPage : 1
+  );
+
+  function handleChange(event) {
+    setSelectedPage(event.target.value);
+    props.onChange &&
+      typeof props.onChange === 'function' &&
+      props.onChange(event);
+  }
+
+  //Decreases number by one on previous button click
+  function handlePrev() {
+    if (selectedPage > 1) {
+      setSelectedPage(selectedPage - 1);
+    }
+  }
+
+  //Increases number by one on next button click
+  function handleNext() {
+    if (selectedPage < count) {
+      setSelectedPage(++selectedPage);
+    }
+  }
+
+  function paginationLabel() {
+    if (count <= 1) {
+      return `${i18n.simplePagination.ofLabel}
+        ${count}
+        ${i18n.simplePagination.pageLabel}`;
+    } else {
+      return `${i18n.simplePagination.ofLabel}
+        ${count}
+        ${i18n.simplePagination.pagesLabel}`;
+    }
+  }
+
+  const disabledPrevTooltip = disabled || selectedPage < 2;
+
+  const disabledNextTooltip = disabled || selectedPage >= count;
+
+  const StyledPrevTooltip = styled(Tooltip)`
+    > div {
+      opacity: ${disabledPrevTooltip ? '0' : ''};
+    }
+  `;
+  const StyledNextTooltip = styled(Tooltip)`
+    > div {
+      opacity: ${disabledNextTooltip ? '0' : ''};
+    }
+  `;
+
+  return (
+    <StyledWrapper
+      {...other}
+      theme={theme}
+      data-testid={testId}
+      disabled={disabled}
+      isInverse={isInverse}
+      ref={ref}
+    >
+      {!hidePreviousButton && (
+        <StyledPrevTooltip content={i18n.pagination.previousButtonLabel}>
+          <NavButton
+            aria-label={i18n.pagination.previousButtonLabel}
+            variant={ButtonVariant.link}
+            color={ButtonColor.secondary}
+            disabled={selectedPage < 2 ? true : disabled}
+            icon={<ArrowBackIcon />}
+            isInverse={isInverse}
+            theme={theme}
+            onClick={handlePrev}
+            shape={ButtonShape.fill}
+          />
+        </StyledPrevTooltip>
+      )}
+      <Spacer size={14} />
+
+      <NativeSelect
+        aria-label={i18n.select.placeholder}
+        data-testid={`${testId}-select`}
+        disabled={disabled}
+        fieldId={id}
+        isInverse={isInverse}
+        onChange={handleChange}
+        value={selectedPage}
+      >
+        {Array.from({ length: count }, (_, i) => (
+          <option
+            data-testid={`${testId}-option`}
```

Comment:
This needs to be unique:
```suggestion
            data-testid={`${testId}-option-${i}`}
```